src/HeaderCell.tsx

```diff
@@ -54,27 +59,32 @@ export default function HeaderCell<R, SR>({
     }
 
     function onPointerMove(event: PointerEvent) {
-      if (event.pointerId !== pointerId) return;
       if (event.pointerType === 'mouse' && event.buttons !== 1) {
-        onPointerUp(event);
+        // handle case where the pointer `up`'d outside an iframe
+        // https://bugs.chromium.org/p/chromium/issues/detail?id=606896
+        // https://bugs.chromium.org/p/chromium/issues/detail?id=693494
+        onPointerUp();
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/setPointerCapture
Actually the example on this page is in an iframe and it works fine, wonder what's different with the storybook.
We could remove this check 🤔